camunda-formio-tasklist-vue/src/components/search/TaskSearchItem.vue

```diff
@@ -1,14 +1,14 @@
 <template>
   <div>
     <b-row>
-    <!-- this is for close the filter on left top side -->
+      <!-- this is for close the filter on left top side -->
```

Comment:
Do run `npm run lint`, and let me know if there is any modifications after that. If yes, update here